ros_buildfarm/templates/status/js/setup.js

```diff
@@ -238,15 +238,24 @@ function filter_table() {
       result_rows = $.map(window.rows, function(row) {
         for (var i = 0; i < queries.length; i++) {
           var is_known_query = false;
+          var is_id_query = false;
           for(var q in QUERY_TRANSFORMS) {
-            if (queries[i] === QUERY_TRANSFORMS[q]) {
+            if (RegExp("^("+QUERY_TRANSFORMS[q]+")$").test(queries[i])) {
```

Comment:
yeah I was testing with it too, the rest of the pages' queries are here https://github.com/dhood/ros_buildfarm/blob/add-blocking-script/ros_buildfarm/templates/status/js/setup.js#L7 but I suppose there is a risk that in the future someone will want to use a query called id for a different purpose